crates/orderbook/src/database/fee_policies.rs

```diff
@@ -0,0 +1,107 @@
+use {
+    anyhow::Context,
+    bigdecimal::{BigDecimal, FromPrimitive},
+    database::{auction::AuctionId, OrderUid},
+    model::fee_policy::{FeePolicy, Quote},
+    number::conversions::big_decimal_to_u256,
+    std::collections::HashMap,
+};
+
+impl super::Postgres {
+    pub async fn fee_policies(
+        &self,
+        keys_filter: &[(AuctionId, OrderUid)],
+    ) -> anyhow::Result<HashMap<(AuctionId, OrderUid), Vec<FeePolicy>>> {
+        let mut ex = self.pool.acquire().await?;
+
+        let _timer = super::Metrics::get()
```

Comment:
isn't it this timer stopped?

Reply:
It is stopped once dropped.

Reply:
Thanks, forgot to stop it after refactoring.